test/drbg_cavs_test.c

```diff
@@ -254,6 +254,11 @@ static int test_cavs_kats(const struct drbg_kat *test[], int i)
     const struct drbg_kat *td = test[i];
     int rv = 0;
 
+#ifdef FIPS_MODE
+    /* FIPS mode doesn't support CTR DRBG without a derivation function */
```

Comment:
This comment applies to all three drbg types. You could replace it by something like

```C
   /* FIPS mode doesn't support instantiating without a derivation function */
```

Reply:
The other two kinds of DRBG always have a DF, so the case is impossible.

Reply:
Comment changed.